.changelog/19955.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+data-source/aws_lb_target_group: Add cookie_name attribute for app_cookie support
```

Comment:
Not a bug before a release 😄.

```suggestion
```release-note:enhancement
data-source/aws_lb_target_group: Add `cookie_name` argument for application cookie stickiness
```